packages/@uppy/status-bar/src/StatusBar.tsx

```diff
@@ -33,16 +43,12 @@ function getUploadingState (error, isAllComplete, recoveredState, files) {
     }
     // If files are being preprocessed AND postprocessed at this time, we show the
     // preprocess state. If any files are being uploaded we show uploading.
-    if (progress.preprocess && state !== statusBarStates.STATE_UPLOADING) {
```

Comment:
is this a safe change?

Reply:
Note that when you comment on the deleted line, GitHub is being annoying because it won't show the added line, and won't let me add a suggestion. Consider commenting the added lines instead.

To answer your question, yes it's safe, it's dead code (you even thumbed up https://transloadit.slack.com/archives/C0FMW9PSB/p1703243841860289)